src/influx_line.erl

```diff
@@ -0,0 +1,626 @@
+%%% @doc
+%%% InfluxDB Line protocol encoder.
+%%% @end
+-module(influx_line).
+-include_lib("influx_udp/include/influx_udp_priv.hrl").
+-include_lib("influx_udp/include/influx_udp.hrl").
+
+-define(Q, <<"\"">>).
+-define(SPACE, <<" ">>).
+-define(NEW_LINE, <<"\n">>).
+-define(EMPTY, <<"">>).
+-define(COMMA, <<",">>).
+-define(EQ, <<"=">>).
+
+-export([
+  encode/1,
+  encode/2,
+  encode/3,
+  encode/4,
+  encode_tags/1,
+  encode_fields/1,
+  timestamp/0
+]).
+
+%% ------------------------------------------------------------------
+%% API Function Definitions
+%% ------------------------------------------------------------------
+
+%% @doc
+%% Encode InfluxDB data point (or list of points) to line.
+%% Point should contain 'measurement' and 'fields' keys (as atoms).
+%% Other available keys: 'tags', 'time'.
+%% Return binary line(-s) or `{error, invalid_data}` if any point contains
+%% neither 'measurement' nor 'fields' key. 
+%%
+%% @example
+%%   influx_line:encode(
+%%     #{
+%%        measurement => cpu,
+%%        fields => #{ value => 43},
+%%        tags => #{ host => 'eu-west', ip => "127.0.0.1" },
+%%        time => 10000000000
+%%      })
+%%   /=> <<"cpu,host=eu-west,ip="127.0.0.1" value=43 10000000000\n">>
+%% @end
+-spec encode(Points::influx_data_points()) -> Line::binary()|{error, Reason::atom()}.
```

Comment:
Line 46 is too long: -spec encode(Points::influx_data_points()) -> Line::binary()|{error, Reason::atom()}..